src/helpers/issue.ts

```diff
@@ -367,103 +315,108 @@ export async function isUserAdminOrBillingManager(
 }
 
 export async function addAssignees(context: Context, issue: number, assignees: string[]) {
-  const payload = context.event.payload as Payload;
+  const payload = context.payload;
   try {
-    await context.event.octokit.rest.issues.addAssignees({
+    await context.octokit.rest.issues.addAssignees({
       owner: payload.repository.owner.login,
       repo: payload.repository.name,
       issue_number: issue,
       assignees,
     });
   } catch (e: unknown) {
-    context.logger.debug("Adding assignees failed!", e);
+    context.logger.error("Adding assignees failed!", e);
   }
 }
 
 export async function deleteLabel(context: Context, label: string) {
-  const payload = context.event.payload as Payload;
+  const payload = context.payload;
   try {
-    const response = await context.event.octokit.rest.search.issuesAndPullRequests({
+    const response = await context.octokit.rest.search.issuesAndPullRequests({
       q: `repo:${payload.repository.owner.login}/${payload.repository.name} label:"${label}" state:open`,
     });
     if (response.data.items.length === 0) {
       //remove label
-      await context.event.octokit.rest.issues.deleteLabel({
+      await context.octokit.rest.issues.deleteLabel({
         owner: payload.repository.owner.login,
         repo: payload.repository.name,
         name: label,
       });
     }
   } catch (e: unknown) {
-    context.logger.debug("Deleting label failed!", e);
+    context.logger.error("Deleting label failed!", e);
```

Comment:
I was thinking of renaming this method to `fatal` and change `warn` to error to be used in most cases.